biz.aQute.tester.test/no-engines.bndrun

```diff
@@ -0,0 +1,22 @@
+# Set javac settings from JDT prefs
+-include: ${workspace}/cnf/includes/jdt.bnd
```

Comment:
The above lines should not be in a bndrun file. Just bnd.bnd.

Reply:
This whole file can go. It is a relic from when this project used to actually contain concrete tests, rather than test utilities.